src/content_script/mastodonInject.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * @typedef {Object} VersionNumber
+ * @property {string} major
+ * @property {string} minor
+ * @property {string} patch
+ */
+
+/**
+ * parses a versionNumber string
+ * @returns {VersionNumber}
+ */
+function parseVersionNumber() {
+  const versionElement = document.querySelector(`.link-footer`).textContent
+  const versionNumber = versionElement.match(/v?(?<major>\d+)\.(?<minor>\d+)\.(?<patch>\d+)$/)
+  return versionNumber.groups
+}
+
+
+/**
+ * Replacement onClick handler for Follow button
+ * @param {Event} event 
+ */
+function onClickFollow (event) {
+  event.stopPropagation()
+  event.preventDefault()
+  const username = window.location.pathname.split(`/`).slice(-1)[0]
+  // activate AutoRemoteFollow
+  window.open(`/users/${username}/remote_follow`, `_blank`)
+}
+
+/**
+ * wait for element to appear
```

Comment:
Ah cool algo, did you copy that somewhere from? Stackoverflow? If so, could you please link the source.

Feel free to do that via [@see](https://jsdoc.app/tags-see.html).